schemas/JSON/settings/settings.schema.0.2.json

```diff
@@ -184,6 +184,11 @@
           "type": "integer",
           "default": 3,
           "minimum": 1
+        },
+        "extractArchiveWithTar": {
```

Comment:
That isn’t the name of the setting -
https://github.com/microsoft/winget-cli/blob/8ac5e07cdd3f401b11a86665bd7281602526acd9/doc/Settings.md#archive-extraction-method